bar/90_contents/src/login.html

```diff
@@ -24,88 +24,7 @@
-                                        $("#userspanid").removeClass('alertSuccessMsg');
-                                        $("#userspanid").addClass('alertmsg');
-                                        $("#ProceedButton").attr("disabled", "disabled");
-                                    } else if (response['userIdMsg'] == "available") {
-                                        msg = uname + " is available";
-                                        $("#userspanid").removeClass('alertmsg');
-                                        $("#userspanid").addClass('alertSuccessMsg');
-                                        $("#ProceedButton").removeAttr("disabled");
-                                    }
-                                    $("#userspanid").html(msg);
-                                }
-                            });
-                    }
-                    else {
-                        $("#userspanid").html("");
-                    }
-                }
-            });
-
-            var errorFlag = null;
-            var showMessage = '';
-            if (errorFlag == 'null' && showMessage == 'null') {
-                document.getElementById("logoutDiv").style.visibility = "hidden";
-            } else {
-                document.getElementById("logoutDiv").style.visibility = "visible";
-                $("#logoutDiv").addClass("loginErrorMessage");
-                $("#logoutMsg").text(showMessage);
-            }
-
-            var user = localStorage ? localStorage.user : null;
-            if (user) {
-                document.getElementById("logoutDiv").style.visibility = "visible";
-                $("#logoutDiv").addClass("logoutSuccess");
-                //var username = sessionStorage.logoutShorterUName;
-                $("#logoutMsg").text("You have logged out successfully.");
-                $('#logoutMsg').attr('title', user);
-                localStorage.user = '';
-            }
-            var browserLanguage = navigator.language;
-            if (browserLanguage != undefined && browserLanguage.toLowerCase() == 'ja') {
-                $("#ddLanguageSelector").val('Japanese(jp)');
-            }
-
-            // If it is not a local file, hide unitURL and unitCellName and extract from location.
-            var cellUrlSplit = _.first(location.href.split("#")).split("/");
-            if (!_.contains(cellUrlSplit, "file:") && !_.contains(cellUrlSplit, "localhost")) {
-                // Hide unitURL and unitCellName
-                $(".dtUnitUrl").toggle(false);
-                $(".dtUnitCellName").toggle(false);
-                $("#loginForm").addClass("localCellManager");
-
-                // Extract rootURL and cell name from location
-                var cellInfo = uLogin.getCellInfo(cellUrlSplit);
-
-                $("#unitUrl").val(cellInfo.rootURL);
-                $("#unitCellName").val(cellInfo.cellName);
-            }
+            uLogin.initCellManager();
```

Comment:
Do you need to perform the same changes to the normal login.html, too?

Reply:
I agree.
Passing arguments to initCellManager makes it possible to do it, and I think that it is unnecessary to split html into two.

Reply:
Since processing to save token is a function in js, it is not necessary to modify login.html of unitCell.
However, since processing in document.ready is a function, I think that it is possible to share it by adding arguments.